client/src/components/DayView/Calendar.js

```diff
@@ -184,24 +184,27 @@ class Calendar extends React.Component {
             fetchEvents={this.fetchRoomEvent}
           />
         )}
-        <DatePicker
-          className="date"
-          value={currentDate}
-          onChange={(value) => {
-            this.setState({ currentDate: value });
-            if (!value) return;
-            const calendarApi = this.calendarComponentRef.current.getApi();
-            calendarApi.gotoDate(value.toISOString(true));
-          }}
-          onClick={() =>
-            this.setState({
-              currentDate: moment(
-                this.calendarComponentRef.current.getApi().getDate()
-              ),
-            })
-          }
-          allowClear={false}
-        />
+        <div className="date-container">
+          <labeel>Go To Date</labeel>
+          <DatePicker
+            className="date"
+            value={currentDate}
```

Comment:
can you add `disabledDate()` attribute to disable `hiddenDays` like Friday and Saturday ?
because user can't select them anyway

```js
<DatePicker
   disabledDate={(current) =>
      hiddenDays.includes(Number(current.format('e')))
   }
/>
```
![image](https://user-images.githubusercontent.com/41539949/84195647-9a1be080-aaa7-11ea-9dd9-5ef9c90f7ccf.png)

Reply:
ok